optuna_dashboard/_app.py

```diff
@@ -447,6 +449,41 @@ def save_trial_note(study_id: int, trial_id: int) -> dict[str, Any]:
         note.save_note_with_version(storage, study_id, trial_id, req_note_ver, req_note_body)
         response.status = 204  # No content
         return {}
+    
+    @app.get("/csv/<study_id:int>")
+    def download_csv(study_id: int) -> BottleViewReturn:
```

Comment:
To strictly follow the logic of this table (ref: https://github.com/optuna/optuna-dashboard/blob/main/optuna_dashboard/ts/components/TrialTable.tsx), it will be easy to use this class. https://github.com/optuna/optuna-dashboard/blob/main/optuna_dashboard/_cached_extra_study_property.py#L43

Reply:
To follow the original logic, there are some differences as follows:

* Displaying user_attrs instead of union_user_attrs
* Displaying params without considering union_search_space and intersection_search_space